docs/api/TestUtils.md

```diff
@@ -0,0 +1,194 @@
+# API: Utils
+
+## MockActionContext
+
+Useful for testing your actions, you can pass an instance of `MockActionContext` to your action to record the methods that the action calls on the context.
+
+When `dispatch` is called, it will push an object to the `dispatchCalls` array. Each object contains a `name` and `payload` key.
+
+When `executeAction` is called, it will push an object to the `executeActionCalls` array. Each object contains an `action` and `payload` key.
+
+`getStore` calls will be proxied to a dispatcher instance, which you can register stores to via `MockActionContext.registerStore(MockStore)`.
+
+### Usage
+
+Here is an example mocha test that display using each of `ActionContext` methods being tested:
+
+```js
+var MockActionContext = require('./utils').createMockActionContext();
```

Comment:
Should we update this to use `require('fluxible/utils').createMockActionContext();` instead of the relative path?